pkg/gardenlet/controller/shoot/care/health.go

```diff
@@ -530,11 +537,23 @@ func (h *Health) checkWorkers(
 	return &c, nil
 }
 
+var constraintGardenerGreaterEqual190 *semver.Constraints
+
+func init() {
+	var err error
+	constraintGardenerGreaterEqual190, err = semver.NewConstraint(">= 1.90-0")
+	utilruntime.Must(err)
+}
+
```

Comment:
Should this eventually be removed again, i.e. do we need a `todo` here?

Reply:
Resolved via https://github.com/gardener/gardener/pull/9313 as discussed in https://github.com/gardener/gardener/pull/9335#discussion_r1517546923.